kolibri/content/files_for_testing/article.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
```

Comment:
Most of the files in this folder do not get referenced.

Please delete them completely from the git history to avoid repo bloat.